app/views/catalog/_expert_help_header.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="collapse-toggle panel-heading expert-box" data-toggle="collapse" data-target="#ExpertOptions">
```

Comment:
I still think we should consider putting in the initial collapsed class here because on the mobile view the arrow moves from pointing down to up due to the javascript load delay. Maybe it's more noticeable because I'm in the dev environment?

Reply:
i don't notice it, but initially adding the class seems logical. :+1: